src/data/components/Autocomple.tsx

```diff
@@ -0,0 +1,100 @@
+import { useCallback, useState } from 'react';
+import { Person } from '../../types/Person';
+import debounce from 'lodash.debounce';
+import classNames from 'classnames';
+
+type Props = {
+  delay?: number;
+  people: Person[];
+  setSelectedPerson: (person: Person) => void;
+};
+
+export const Autocomplete: React.FC<Props> = ({
+  delay = 300,
+  people,
+  setSelectedPerson,
+}) => {
+  const [query, setQuery] = useState('');
+  const [aplliedQuery, setApliedQuery] = useState('');
+  const [focus, setFocus] = useState(false);
+
+  const aplyQuery = useCallback(debounce(setApliedQuery, delay), []);
```

Comment:
Fix it 
<img width="964" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/61921bd9-a0b0-41b1-90e5-3ffebe0f77a0">